### PR TITLE
Add getGlobalAmpleforthEpochAndAMPLSupply

### LINKED ISSUE #188: add getter on policy which returns epoch and global supply

### PR DESCRIPTION
**Pull request by nithinkrishna**
Added a getter on the Policy contract which returns the `globalAmpleforthEpoch` and `globalAMPLSupply`. The MonetaryPolicy (on the base chain) and the XCAmpleController on the satellite chains will implement this method.

When rebase is propagated between chains, its these 2 values which are transmitted. The method allows consumers to read both values atomically.  

fixes #188

### CODE REVIEW

File: contracts/UFragmentsPolicy.sol [hunk 1]
```diff
@@ -210,6 +210,20 @@ contract UFragmentsPolicy is Ownable {
...
+     * @notice A multi-chain AMPL interface method. The Ampleforth monetary policy contract
+     *         on the base-chain and XCAmpleController contracts on the satellite-chains
+     *         implement this method. It returns what the current contract believes to be
+     *         the globalAmpleforthEpoch and globalAMPLSupply.
+     * @return globalAmpleforthEpoch The current epoch number.
+     * @return globalAMPLSupply The total supply at the current epoch.
+     */
+    function getGlobalAmpleforthEpochAndAMPLSupply() external view returns (uint256, uint256) {
+        uint256 globalAmpleforthEpoch = epoch;
+        uint256 globalAMPLSupply = uFrags.totalSupply();
```
Comment (brandoniles) [inline contracts/UFragmentsPolicy.sol:223]: Is this really more efficient than doing...
`return (epoch, uFrags.totalSupply());`

File: contracts/UFragmentsPolicy.sol [hunk 2]
```diff
@@ -210,6 +210,20 @@ contract UFragmentsPolicy is Ownable {
         rebaseWindowLengthSec = rebaseWindowLengthSec_;
     }
 
+    /**
+     * @notice A multi-chain AMPL interface method. The Ampleforth monetary policy contract
+     *         on the base-chain and XCAmpleController contracts on the satellite-chains
+     *         implement this method. It returns what the current contract believes to be
```
Comment (brandoniles) [inline contracts/UFragmentsPolicy.sol:216]: "It returns what the...." 
->
"It atomically returns two values: what the current..."

**Review by brandoniles — APPROVED** [review]

File: contracts/UFragmentsPolicy.sol (hunk 1 above)
Comment (nithinkrishna) [reply contracts/UFragmentsPolicy.sol:223]: 👍

File: contracts/UFragmentsPolicy.sol (hunk 2 above)
Comment (nithinkrishna) [reply contracts/UFragmentsPolicy.sol]: 👍

**Pull request merged**